streaming_data_types/utils.py

```diff
@@ -24,3 +25,36 @@ def check_schema_identifier(buffer, expected_identifer: bytes):
         raise WrongSchemaException(
             f"Incorrect schema: expected {expected_identifer} but got {get_schema(buffer)}"
         )
+
+
+def latest_schema(schema_type: str):
+    """
+    Returns the latest schema identifier for that type of schema
+    """
+    return all_schemas()[schema_type]
+
+
+def all_schemas():
+    """
+    Returns list of all schemas
+    """
+    fbs_identifier = {
+        "alarm": "al00",
+        "area_detector": "ad00",
+        # "array_1d": "se00", # deprecated for sample_environment
+        "dataarray": "da00",
+        "epics_connection": "ep01",
+        "eventdata": "ev44",
+        "finished_writing": "wrdn",
+        "forwader": "fc00",
+        # "histogram": "hs01", # deprecated for dataarray
```

Comment:
No, it is not. I feel like I constantly saying this :D

Reply:
Ok!  I will sticky-note it to my desk this morning!  Changed the behaviour